src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+a,
+h1,
+h2,
+h3,
+p,
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+.header {
+  margin: 0 50px;
+  padding: 10px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. This is a good practice to specify a generic font family in case the main one doesn't work. For example, you could use: `font-family: Roboto, Arial, sans-serif;`